lib/plugin.rb

```diff
@@ -1,35 +1,47 @@
 require 'cocoapods-core'
 
 module CocoaPodsKeys
-  include Pod::Config::Mixin
+  class << self
+    def podspec_for_current_project(spec_contents)
+      keyring = KeyringLiberator.get_keyring_named(user_options["project"]) || KeyringLiberator.get_keyring(Dir.getwd)
+      raise Informative, "Could not load keyring" unless keyring 
+      key_master = KeyMaster.new(keyring)
+
+      spec_contents.gsub!(/%%SOURCE_FILES%%/, "#{key_master.name}.{h,m}")
+      spec_contents.gsub!(/%%PROJECT_NAME%%/, user_options["project"])
+    end
+
+    def setup
+      require 'preinstaller'
+
+      PreInstaller.new(user_options).setup
+      # Add our template podspec (needs to be remote, not local).
+      podfile.pod 'Keys', :git => 'https://github.com/ashfurrow/empty-podspec.git'
+    end
+
+    private
+
+    def self.podfile
```

Comment:
I had to remove `self.` for `podfile` and `user_options`, otherwise I got errors about undefined symbols.